### PR TITLE
Add Source Sans Pro to styles

### LINKED ISSUE #460: Re-add Source Sans Pro
Somehow the code for this font was deleted. Need to re-add it back to the website as the fallback Verdana is far too large in comparison to the headings that are all in Montserrat. 

Need to update Verdana to Arial.

### PR DESCRIPTION
**Pull request by ststimac**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

## Short description of the change(s)

Fix #460
The code to Source Sans Pro was removed at some point but it is still
defined as the main font for the body of the site.

Added Arial in replacement of Verdana for a fallback font since it is
proportionally very different from Source Sans Pro and the current type
scale makes Verdana seem very large compared to the headings.

### CODE REVIEW

File: src/sonarwhal-theme/layout/partials/styles.hbs [hunk 1]
```diff
@@ -16,6 +16,5 @@
 <link rel="stylesheet" href="/core/css/controls.css">
 <link rel="stylesheet" href="/core/css/scanner.css">
 <!-- endbuild -->
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro"> 
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500" integrity="sha384-6hCj8YZdGj+SQ9wB8ShBR1XRd+tUsQSfKu1i4C72EaUKO6AeEipi63+OPsdZPLoD" crossorigin="anonymous">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500|Source+Sans+Pro" integrity="sha384-6hCj8YZdGj+SQ9wB8ShBR1XRd+tUsQSfKu1i4C72EaUKO6AeEipi63+OPsdZPLoD" crossorigin="anonymous">
```
Comment (alrra) [inline src/sonarwhal-theme/layout/partials/styles.hbs]: > sha384-KfN8k2PKf2Ze/0sCe/2VHfHet5sTnHszVhdKaRwov2a9VdEZJJfvc7gZxaZbBl1J

**Comment by ststimac** [comment]
I was able to get down to 3 font weights for Montserrat. There's a light, medium and heavy weight font. I tried interchanging the medium and light fonts to see if I could combine into one, but the type size for lines using the light weight vs the medium weight are vastly different.

**Comment by molant** [comment]
Can you rebase from master instead of merging?

**Comment by ststimac** [comment]
@molant done.

File: src/sonarwhal-theme/layout/partials/styles.hbs [hunk 2]
```diff
@@ -16,5 +16,5 @@
 <link rel="stylesheet" href="/core/css/controls.css">
 <link rel="stylesheet" href="/core/css/scanner.css">
 <!-- endbuild -->
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500" integrity="sha384-6hCj8YZdGj+SQ9wB8ShBR1XRd+tUsQSfKu1i4C72EaUKO6AeEipi63+OPsdZPLoD" crossorigin="anonymous">
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.css" integrity="sha384-mHx5KeR3zcLviQFPmpHqrwf2uxtukRvUFVzOMOqIjKjSNrurxlmJ3oS+vm49Mz07" crossorigin="anonymous">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500|Source+Sans+Pro" integrity="sha384-KfN8k2PKf2Ze/0sCe/2VHfHet5sTnHszVhdKaRwov2a9VdEZJJfvc7gZxaZbBl1J" crossorigin="anonymous">
```
Comment (alrra) [inline src/sonarwhal-theme/layout/partials/styles.hbs]: > sha384-VBLZm6c9Klc07mT9pV/Gy4zZFk8EI4bwWJMcKnInnKaT2QRvQa39WIak0FIQqb/M

Will change it before merging.

**Pull request closed without merging**